packages/manager/apps/dedicated/client/app/network-security/traffic/traffic.routing.js

```diff
@@ -14,6 +16,8 @@ export default /* @ngInject */ ($stateProvider) => {
       resolve: {
         breadcrumb: /* @ngInject */ ($translate) =>
           $translate.instant('network_security_dashboard_traffic_breadcrumb'),
+        ip: /* @ngInject */ ($state) => () => $state.params.ip,
+        dateTime: /* @ngInject */ ($state) => () => $state.params.dateTime,
```

Comment:
My bad @Steffy29 , I made a small mistake in the last review, no need to declare a function and you can use directly `$stateParams`
```suggestion
        ip: /* @ngInject */ ($stateParams) => $stateParams.ip,
        dateTime: /* @ngInject */ ($state) => $stateParams.dateTime ? new Date($stateParams.dateTime) : null,
```